hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/s3/tenant/OMTenantRequestHelper.java

```diff
@@ -94,26 +99,26 @@ static void checkTenantExistence(OMMetadataManager omMetadataManager,
    * Retrieve volume name of the tenant.
    */
   static String getTenantVolumeName(OMMetadataManager omMetadataManager,
-      String tenantName) {
+      String tenantId) {
 
     final OmDBTenantInfo tenantInfo;
     try {
-      tenantInfo = omMetadataManager.getTenantStateTable().get(tenantName);
+      tenantInfo = omMetadataManager.getTenantStateTable().get(tenantId);
     } catch (IOException e) {
       throw new RuntimeException("Potential DB error. Unable to retrieve "
-          + "OmDBTenantInfo entry for tenant '" + tenantName + "'.");
+          + "OmDBTenantInfo entry for tenant '" + tenantId + "'.");
```

Comment:
This IOException should be propagated to the caller (validateAndUpdateCache) since it indicates a problem reading from the DB.

Reply:
`OMClientRequest#validateAndUpdateCache` doesn't really propagate any exception.

To throw IOException in this helper method I'd have to add extra try logic in `validateAndUpdateCache` to handle it

Reply:
done

Reply:
Yup moving the call inside the try/catch on applyTransaction looks good. The call to `createErrorOMResponse` when the IOException is caught in applyTransaction will assign the response a status of `INTERNAL_ERROR` and terminate the OM as expected on DB failure.